ractor_actors/src/net/tcp/session.rs

```diff
@@ -112,10 +156,15 @@ where
 
         // let (read, write) = stream.into_split();
         // spawn writer + reader child actors
-        let (writer, _) =
-            Actor::spawn_linked(None, SessionWriter, write, myself.get_cell()).await?;
+        let (writer, _) = Actor::spawn_linked(
+            Some(format!("{}-rw", myself.get_name().unwrap_or_default())),
```

Comment:
forcing named actors means a host of multiple actors of this type could just crash due to name clashes. naming is globally unique.

Reply:
I didn't realize that they needed to be unique, I just used the name for logging myself.